app/subscriber/src/features/my-reports/view/ViewReportInstance.tsx

```diff
@@ -18,14 +18,14 @@ const ReportInstancePreview: React.FC = () => {
     async (reportId: number) => {
       try {
         setIsLoading(true);
-        const response = await viewReportInstance(reportId);
+        const response = await previewReport(reportId);
```

Comment:
Doesn't make sense again.  What's the purpose of this component page?  Is it to "view" an instance or "preview" a report?